notebooks/a00_introduction_to_geomstats.ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<p>We say: "Many datasets lie on a manifold", but we have not introduced a manifold yet. (since it will be done in the next notebook).</p><p> </p><p>Should we still say something like: “the definition of manifold will be introduced later, but for now we can see it as a nonlinear generalization of a vector space, such as the surface in the figure below [add one more beautiful drawing?]”</p><p> </p><p>1) making computations less complicated --&gt; making computations more intuitive and interpretable.</p><p>(Some could argue that Riemannian operations are quite complicated!)</p><p> </p><p>2) predictive power is good actually!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I changed "may give more predictive power" to "will give more predictive power". I think that is what you were saying in (2).</p>

Reply:
<p>Riemannain -&gt; Riemannian</p><p>(This happens several times in the notebooks, I'll let you check them all).</p><p> </p><p>NIT: I would add key words at the beginning of each point to summarize/emphasize the advantage.</p><p> </p><ol><li>Better memory efficiency: Analyzing data on the manifold it lies on allows you to reduce the degrees of freedom of the system, which makes computations less complicated and more intuitive and interpretable.</li><li>Better understanding: Knowing the manifold that a data set belongs to will give you a better understanding of the data's evolution.</li><li>Better prediction: Knowing the manifold a data set lives in give you better predictive power and will help you extract the "signal" from a noisy data set or a data set with very few datapoints.</li></ol><p> </p><p> </p>